### ISSUE TITLE
[Error] No error thrown when require invalid module on Linux platform

### ISSUE THREAD

**Issue by qiaojingx** [01org/zephyr.js#735]
When running [test-error.js](https://github.com/01org/zephyr.js/blob/master/tests/test-error.js) on Linux platform with commit 3f8eb85 via command `zephyr.js$ ./outdir/linux/release/jslinux tests/test-error.js` 

Expected result:  Error is thrown when require invalid module just like on Arduino 101 and FRDM-K64F.

Actually result: No error was thrown when require invalid module, only got error log.

Serial log:
```
Testing error types...

PASS - SecurityError works as expected
PASS - NotSupportedError works as expected
PASS - SyntaxError works as expected
PASS - TypeError works as expected
PASS - RangeError works as expected
PASS - TimeoutError works as expected
PASS - NetworkError works as expected
PASS - SystemError works as expected
PASS - calling require with an integer
PASS - calling require with a long name

src/zjs_script.c:17 zjs_read_script():
[ERROR] error opening file

src/zjs_modules.c:154 native_require_handler():
[ERROR] could not read module modules/easterbunny
calling require with an invalid module name: expected error object
FAIL - calling require with an invalid module name
TOTAL: 10 of 11 passed
```

Branch | Target Device | Build ID | Date | Result
---|---|---|---|---
master | Linux | 21915c0 | Feb 7, 2017 | FAIL by 677 
master | Linux | 3f8eb85 | Feb 13, 2017 | FAIL

**Comment by grgustaf** [comment]
This should be fixed by #774.

**Comment by qiaojingx** [comment]
Verified with commit b29dbbc. This issue is not reproducible on Linux. Closed.
